buddyboss/buddyboss-platform
# #706 Fix emoji not working in group messages

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #706 .

### How to test the changes in this Pull Request:

1. Go to  group
2. Click on Send Message Tab
3. Send the Message to users and make sure message contain emoji added
4. Go to Message and you will get the emoji added on message content.


### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
